app/controllers/services/govbox_controller.rb

```diff
@@ -84,4 +86,21 @@ def ensure_live
       render template: 'errors/not_found', status: 404, layout: 'application'
     end
   end
+
+  def fetch_statutory_entries
+    response = RestClient::Resource.new("#{ENV.fetch('AUTOFORM_URL')}/api/corporate_bodies/search", timeout: 10).get(params: { q: "cin:#{params[:cin]}", private_access_token: ENV.fetch('AUTOFORM_PRIVATE_ACCESS_TOKEN') })
```

Comment:
Tomuto by sa mohlo dat cisto teoreticky vyhnut, kedze autoform to vrati v JS response v predchadzajucom kroku nie?

Reply:
statutarov sme do JS nepridavali, myslim ze to teda potrebujeme tahat cez rest

Reply:
@michal-rohacek myslim, ze sme to tam pridavali. Teda tusim, ze ty osobne :D https://github.com/slovensko-digital/autoform.ecosystem/pull/55

Reply:
@jsuchal jasne, pamatam si to, lenze ak tomu spravne rozumiem - v predoslom kroku pouzivame autocomplete, a teda ak by sme tie data chceli posunut do dalsieho kroku, tak by sme ich asi chceli pridat do nejakych hidden fields, a to tiez cez autocomplete. lenze do autocomplete sme statutarov nepridavali:
https://github.com/slovensko-digital/autoform.ecosystem/blob/master/app/views/assets/autoform.js.erb#L48-L66
ci to myslis inak a ja nerozumiem? :D

Reply:
@michal-rohacek hej, myslel som to tak, ze by sme pridali nejaky hidden field na cely ten vybrany search result ako nejaky surovy JSON, ale asi je to trosku nebezpecne, kedze by tam mohol niekto nieco podstrcit cez JS. Takze nechajme takto.